parlai/core/teachers.py

```diff
@@ -1391,6 +1391,18 @@ def _setup_data(self, path):
                         f"for you automatically. This is happening on Line {line_no} "
                         f"in {path}. The line is:\n\t{line}"
                     )
+                if msg and 'text' not in msg:
```

Comment:
A thought, image chat / captioning task does not have text, in this case, they should be None or ''?

Reply:
They probably wouldn't use the fromfile teacher.